README.md

```diff
@@ -2,57 +2,65 @@
 
 ## Description
 
-The application is a clone of [freeCodeCamp's 30 most recent forum posts](https://forum.freecodecamp.org/latest). It is developed by the members of the freeCodeCamp community in order to develop soft skills, get more familiar with Fetch API, Git workflow and develop general coding skills. _And also, just for the sake of happy coding!_
+This work is inspired by the [freeCodeCamp Forum Homepage](https://www.freecodecamp.org/learn/coding-interview-prep/take-home-projects/build-a-freecodecamp-forum-homepage) project. It is a standalone front-end application and a clone of [freeCodeCamp's 30 most recent forum posts](https://forum.freecodecamp.org/latest) page built with ![HTML5](https://img.shields.io/badge/html5-%23E34F26.svg?style=for-the-badge&logo=html5&logoColor=white)![CSS3](https://img.shields.io/badge/css3-%231572B6.svg?style=for-the-badge&logo=css3&logoColor=white)![JavaScript](https://img.shields.io/badge/javascript-%23323330.svg?style=for-the-badge&logo=javascript&logoColor=%23F7DF1E). It is developed by the members of the freeCodeCamp community in order to develop soft skills, get more familiar with Fetch API, Git workflow and develop general coding skills. _And also, just for the sake of happy coding!_
```

Comment:
The placement of these badges seems out of place to me.
<img width="682" alt="Screen Shot 2022-09-29 at 12 52 28 PM" src="https://user-images.githubusercontent.com/67210629/193128765-9fe6f9c8-bb9e-4a1f-9737-020fbd5fdfe1.png">

If we are going to use badges, maybe it would be better to have it in its own section like tech stack. 
<img width="368" alt="Screen Shot 2022-09-29 at 12 56 49 PM" src="https://user-images.githubusercontent.com/67210629/193129534-e8e433d3-f977-43dd-af20-a79ca9e22309.png">

Reply:
You're right, it looks better in separate list.